standalone-tools/wss-scan.sh

```diff
@@ -0,0 +1,126 @@
+###### Information ############################################################################
+# Name:          wss-scan.sh
+# Maintainer:    ODFE Infra Team
+# Language:      Shell
+#
+# About:         This script is to scan the odfe distros for vulnerabilities and licenses
+#                It will scan the repositories and send the WhiteSource link to the mail 
+#                of the user. 
+#
+# Prerequisites: Need to install Java 14
+#                Export JAVA_HOME env variable to the JDK path
+#		 Add JAVA_HOME to PATH variable
+#                Need to set the recepient mail in wss-scan.config for local run
+#                API token is needed for local run
+#
+# Usage:         ./wss-scan.sh
+#
+# Starting Date: 10-13-2020
+# Modified Date: 10-15-2020
+###############################################################################################
+
+set -e
+
+java -version
+if [ $? != 0 ] ; then echo "Java has not been setup" ; exit ; fi
+
+# download the WhiteSource Agent 
+curl -LJO -sS https://github.com/whitesource/unified-agent-distribution/releases/latest/download/wss-unified-agent.jar
+
+# The version 20.9.2.1 has been tested and can be used if a specific version is required
+#curl -LJO -sS https://github.com/whitesource/unified-agent-distribution/releases/download/v20.9.2.1/wss-unified-agent.jar
+
+
+echo "wss-unified-agent jar download has been completed"
+
+# scan the config file for the user configurations
+# wss-scan.config has to be present in the same working directory as the script
+source wss-scan.config
+
+# change comma to whitespace
+gitRepos=${gitRepos//,/$'\n'} 
+
+basepath=$baseDirPath"/repos"
+
+if [[ ! -e $basepath ]]; then
```

Comment:
[[ ]] is a bash build-in, non POSIX.
And you didnt specify a shebang for `#!/bin/bash` at the top of the script.
This is very dangerous and wont work in non-bash compatible supersets.

Reply:
Considering always use POSIX compliant methods like `[ ! -e "$basepath" ]`

Reply:
I was assuming that the git action specifically add /bin/bash at the start of the run. Correct me if I am wrong.

Reply:
I thought you are planning to run this script both local and on GitHub actions.
If so, you cannot assume everything is bash wrapped execution.
Even if GitHub claims they use bash to call, you should always have a fallback.

Reply:
My own machine at home strictly use Bounce Shell as default instead of bash.
This will for sure break on my own machine.

Reply:
Considering adding shebang as a must for all the script to strictly define your runtime env.

Reply:
Yeah, this needs a shebang. This script is not tied to Github Actions.